python/pyarrow/_fs.pyx

```diff
@@ -260,6 +354,44 @@ cdef class FileSelector(_Weakrefable):
         If true, an empty selection is returned.
     recursive : bool, default False
         Whether to recurse into subdirectories.
+
+    Examples
+    --------
+    Generate a file:
+
+    >>> from pyarrow import fs
+    >>> local = fs.LocalFileSystem()
+    >>> with local.open_output_stream('/tmp/fileinfo.dat') as stream:
+    ...     stream.write(b'data')
+    ...
+    4
+
+    Create new directory and subdirectory with copied data:
+
+    >>> # directory
+    >>> local.create_dir('/tmp/alphabet')
+    >>> local.copy_file('/tmp/fileinfo.dat', '/tmp/alphabet/fileinfo.dat')
+    >>> # subdirectory
+    >>> local.create_dir('/tmp/alphabet/aeiou')
```

Comment:
I understand that "aeiou" are the vowels of the alphabet ;) But the first time I read it I found it a bit hard to read (before realizing that it were just all vowels), so maybe something more boring like "subdir" can make it easier to read?

Reply:
It is a good vocal practice :P
Sure, will change.